assets/prototype/domain/eventEditor/containers/relations/useRelationsManager.js

```diff
@@ -5,82 +5,81 @@ import set from 'lodash/set';
 import cloneDeep from 'lodash/cloneDeep';
 
 const useRelationsManager = ({ eventId }) => {
-
 	const [state, dispatch] = useReducer(relationsReducer, {});
 
-	const onReceiveRelations = ({ eventRelations = '{}' }) => {
-		dispatch({ type: 'SET_DATA', data: JSON.parse(eventRelations) });
+	const onReceiveRelations = ({ relations = '{}' }) => {
```

Comment:
That has to be `eventRelations` instead of `relations` because that's the object property in the returned data.